jwt/__init__.py

```diff
@@ -116,7 +117,7 @@ def verify_jwt(jwt,
     """
     header, claims, sig = str(jwt).split('.')
 
-    header = jws.utils.from_base64(header).decode()
+    header = jws.utils.from_base64(header).decode(encoding)
```

Comment:
So `jws.utils.from_base64` ends up calling  `urlsafe_b64decode`, which will return a byte string, which needs to be decoded.
Docs for `decode` on Python 3 say encoding is UTF-8 by default.
However, for Python 2, it says "encoding defaults to the default string encoding", which is normally ascii as you say.
So we definitely need to specify UTF-8 here - thanks.